index.js

```diff
@@ -145,28 +150,28 @@ function formatValue(value) {
     // we then remove the whole wrapping
     // otherwise, the element would be surrounded by quotes: <div a={{b: '<div />'}} />
     return '<__reactElementToJSXString__Wrapper__>' +
-      toJSXString({ReactElement: value, inline: true}) +
+      toJSXString({ReactElement: value, inline: true, options}) +
```

Comment:
as mentioned in the PR description, we need to pass around the `options` object in all these methods only to be able to pass it here (and below in the `recurse` function. I think it may be clearer to access it from the parent scope by moving all these functions inside the body of `toJSXString`